.whitesource

```diff
@@ -0,0 +1,15 @@
+{
+  "scanSettings": {
+    "configMode": "AUTO",
+    "configExternalURL": "",
+    "projectToken": "",
```

Comment:
This needs to be discussed as we need to provide a token but this is an open source project 

@tomekkolo @TomasVrabel